src/plone/restapi/deserializer/dxfields.py

```diff
@@ -169,9 +169,10 @@ def __call__(self, value):
             # Create an ISO 8601 datetime string and parse it with Zope's
             # DateTime module and then convert it to a timezone naive time
             # in local time
-            value = DateTime(u'2000-01-01T' + value).toZone(DateTime(
-            ).localZone()).asdatetime().replace(tzinfo=None).time()
-        except (SyntaxError, DateTimeError):
+            # TODO: should really a timezone naive time be returned?
+            # using ``timetz()`` would be timezone aware.
+            value = dateutil.parser.parse(value).time()
```

Comment:
@buchi @sneridagh @tisto I'm not sure, if time should always converted to timezone-naive. Pythons datetime.time and thus zope.schema Time field supports the tzinfo objects.